client/packages/invoices/src/InboundShipment/DetailView/SidePanel/PricingSection.tsx

```diff
@@ -40,6 +41,11 @@ export const PricingSectionComponent = () => {
     pricing?.serviceTotalAfterTax
   );
 
+  const disableServiceTax =
+    serviceLines
```

Comment:
I thought we are disabling the `stock charge > Tax pen icon` if there ain't any stock charges but here it seems to me we are disabling the Service charges > Tax pen icon.

Reply:
I was confused since issue title said stock but issue itself said `2. Add service charge item`, will change (: